hydromt/workflows/forcing.py

```diff
@@ -248,29 +330,95 @@ def pet(
+    ds = ds.raster.reproject_like(dem_model, method=reproj_method)
+
+    # Process bands like 'pressure' and 'wind'
     if press_correction:
-        ds_out["press"] = press(
+        ds["press"] = press(
             ds["press_msl"],
             dem_model,
             lapse_correction=press_correction,
             freq=None,  # do not change freq of press, put pet_out later
             reproj_method=reproj_method,
         )
     else:
-        ds_out = ds_out.rename({"press_msl": "press"})
+        if "press_msl" in ds:
+            ds = ds.rename({"press_msl": "press"})
+        elif HAS_PYET:
+            # calculate pressure from elevation [kPa]
+            ds["press"] = pyet.calc_press(dem_model)
+            # convert to hPa to be consistent with press function calculation:
+            ds["press"] = ds["press"] * 10
+        else:
+            raise ModuleNotFoundError(
+                "If 'press' is supplied and 'press_correction' is not used, the pyet package must be installed."
+            )
+
     timestep = to_timedelta(ds).total_seconds()
     if method == "debruin":
         pet_out = pet_debruin(
             temp,
-            ds_out["press"],
-            ds_out["kin"],
-            ds_out["kout"],
+            ds["press"],
+            ds["kin"],
+            ds["kout"],
             timestep=timestep,
         )
-    if method == "makkink":
-        pet_out = pet_makkink(temp, ds_out["press"], ds_out["kin"], timestep=timestep)
+    elif method == "makkink":
+        pet_out = pet_makkink(temp, ds["press"], ds["kin"], timestep=timestep)
+    elif "penman-monteith" in method:
+        logger.info("Calculating Penman-Monteith ref evaporation")
+        # Add wind
+        # compute wind from u and v components at 10m (for era5)
+        if ("wind10_u" in ds.data_vars) & ("wind10_v" in ds.data_vars):
+            ds["wind"] = wind(
+                da_model=dem_model,
+                wind_u=ds["wind10_u"],
+                wind_v=ds["wind10_v"],
+                altitude=wind_altitude,
+                altitude_correction=wind_correction,
+            )
+        else:
+            ds["wind"] = wind(
+                da_model=dem_model,
+                wind=ds["wind"],
+                altitude=wind_altitude,
+                altitude_correction=wind_correction,
+            )
+        if method == "penman-monteith_rh_simple":
+            pet_out = pm_fao56(
+                temp["temp"],
+                temp["temp_max"],
+                temp["temp_min"],
+                ds["press"] / 10,
+                ds["kin"],
+                ds["wind"],
+                ds["rh"],
+                dem_model,
+                "rh",
+            )
+        elif method == "penman-monteith_tdew":
+            pet_out = pm_fao56(
+                temp["temp"],
+                temp["temp_max"],
+                temp["temp_min"],
+                ds["press"] / 10,
```

Comment:
temp variable contains the downscaled temperature

Reply:
Yes.